### ISSUE TITLE
A separate command for init and test at the same time without creating any other folder

### ISSUE THREAD

**Issue by Abhishek-dev2** [kp96/nodeforces#34]
I was asked to raise this issue on Reddit under my suggestion in [comments](https://www.reddit.com/r/pcontests/comments/6dub9t/i_have_developed_a_npm_package_for_codeforces/dno1k6n/).

If I have a code written in a file, say `123A.cpp` then there should be a command like `nodeforces run 123A.cpp` which will parse the input and output and run the program. No folder created. The only drawback is that it will parse the test cases every time we run the command.

I will try to add this feature myself but I am not a trustable guy for tasks like these. Will try.

**Comment by kp96** [comment]
Why not cache the problem data in the home folder. This way you can keep the directory clean and there's no need to reparse everytime. Thoughts?

**Comment by Abhishek-dev2** [comment]
Yes, I just wanted my home directory clean. But we should consider both option I guess.

A better idea: Parse in the home folder. If we run the command for the same problem again, then use this folder in the home directory. When we run the command for any new problem, then parse for this new problem and overwrite the folder for the old problem.

We can make a separate folder in the home directory with the name nodeforces and following content -> (inputs cache), (outputs cache), (last submitted problem info), (command log, with results). This can be extended further.

**Comment by kp96** [comment]
Perfect. Instead of overwriting we can create a folder inside home directory (.nodeforces) which is a general norm. This would help keep track of all problem info which we may need in building more features

**Comment by Abhishek-dev2** [comment]
Yes, awesome.
